app/sampl/settings.py

```diff
@@ -20,76 +21,78 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '7&gz49!+qmxwltqo*!_g$n+i)%qcn9c^%2kbwzlnwiuofj29!%'
+SECRET_KEY = "7&gz49!+qmxwltqo*!_g$n+i)%qcn9c^%2kbwzlnwiuofj29!%"
```

Comment:
Just going to make this issue so we remember to change this when we deploy (we should handle it like we do the DB connection)
https://github.com/samplchallenges/SAMPL-league/issues/20

Reply:
We discussed the solution, will be in update to PR